src/main/java/se/liu/ida/hefquin/engine/queryproc/impl/optimizer/evolutionaryAlgorithm/TerminateByDistancePercBest.java

```diff
@@ -0,0 +1,47 @@
+package se.liu.ida.hefquin.engine.queryproc.impl.optimizer.evolutionaryAlgorithm;
+
+import java.util.List;
+
+public class TerminateByDistancePercBest implements TerminationCriterion{
```

Comment:
Can you please include a JavaDoc comment for each termination criterion with a brief description of the criterion.